robotcontainer.py

```diff
@@ -91,9 +94,8 @@ def __init__(self) -> None:
             "elevatorAlgaeHigh", ElevatorAlgaeHigh(self.elevator)
         )
         NamedCommands.registerCommand(
-            "elevatorIntake", ElevatorIntakePosition(self.elevator)
+            "intakeCoral", IntakeCoralProcess(self.elevator, self.intake)
```

Comment:
this should fail, you are passing elevator before intake, read what the class is

Reply:
ye this was branched off before it was fixed -Ivan